lib/state/appActions.js

```diff
@@ -817,4 +825,24 @@ appActions.reset = function(uploadIndex) {
   });
 };
 
+appActions.acknowledge = function (key) {
+  var update = localStore.getItem('acknowledgments');
+
+  if (!update) {
+    update = {};
+  }
+  
+
+  if (!update[key]) {
+    update[key] = {};
+  }
+
+  update[key][this.app.state.targetId] = Date.now();
```

Comment:
Does `Date.now()` stringify to an ISO string automatically? If not, I'd rather explicitly store an ISO string or a `valueOf()` integer.